### PR TITLE
DIKO rewards can be staked immediately

### PR DESCRIPTION
**Pull request by nieldeckx**

### CODE REVIEW

File: clarity/contracts/arkadiko-stake-registry-v1-1.clar [hunk 1]
```diff
@@ -109,6 +109,21 @@
   )
 )
 
+;; Claim pool rewards and stake immediately
+(define-public (stake-pending-rewards 
+  (registry-trait <stake-registry-trait>) 
+  (pool-trait <stake-pool-trait>) 
+  (diko-pool-trait <stake-pool-trait>)
+  (diko-token-trait <ft-trait>)
+  )
```
Comment (philipdesmedt) [inline clarity/contracts/arkadiko-stake-registry-v1-1.clar:118]: REQ: fix indentation

**Review by philipdesmedt — CHANGES_REQUESTED** [review]
LGTM - just fix the indentation and I'll do another functional test and add a button to the new dropdowns

**Comment by nieldeckx** [comment]
@philipdesmedt fixed indentation and added buttons to the new dropdowns

**Comment by philipdesmedt** [comment]
@nieldeckx nice! I'll test

**Review by philipdesmedt — APPROVED** [review]

**Pull request merged**